products/template/build_scripts/transform.py

```diff
@@ -37,6 +37,7 @@ def _invoke_dbt(run_args=list[str]):
 
 
 def staging():
+    _invoke_dbt(["test", "--select", "source:*"])
```

Comment:
came here to say I like new fn names 👏

Reply:
this is testing the source tables, which were created during data loading